### ISSUE TITLE
use different subnet for control plane

### ISSUE THREAD

**Issue by venkataanil** [redhat-performance/jetpack#158]
Currently we use 192.168.24.0/24 subnet for control plane. But undercloud installation is failing If I use different subnet, for example, 192.168.25.0/24. I tried adding below parameters in undercloud.conf
undercloud_public_host = 192.168.25.2
undercloud_admin_host = 192.168.25.3
but still undercloud installation failed.
 Need to check what other parameters we need to change to make it for work.

**Comment by pradiptapks** [comment]
Here is the reference parameter used OSP16 scale test: 

<< From Perf1 NFS share >>
http://perf1.xxxx.xxx.xxx.xxx.redhat.com/pub/psahoo/osp16_scale_test_data/sosreport-f05-h01-000-1029p-osp16-scale-undercloud-2020-02-22-mepsgfg/home/stack/undercloud.conf

**Comment by asyedham** [comment]
@venkataanil I'm able to install undercloud using different subnet like 192.168.25.0/24. I changed the following variables in group_vars/all.yml
```
cidr: 192.168.25.0/24
gateway: 192.168.25.1
dhcp_start: 192.168.25.5
dhcp_end: 192.168.25.105
inspection_iprange: 192.168.25.110,192.168.25.250```

Here you see the parameters are updated with different subnet instead of 192.168.24.0 

undercloud.conf
----------------------
[DEFAULT]
local_interface = p2p4
local_ip = 192.168.25.1/24
undercloud_ntp_servers = clock.redhat.com,clock2.redhat.com
hieradata_override = /home/stack/hiera_override.yaml

[ctlplane-subnet]
cidr = 192.168.25.0/24
gateway = 192.168.25.1
dhcp_start = 192.168.25.5
dhcp_end = 192.168.25.105
inspection_iprange = 192.168.25.110,192.168.25.250

stackrc
-----------
# Clear any old environment that may conflict.
for key in $( set | awk '{FS="="}  /^OS_/ {print $1}' ); do unset $key ; done
NOVA_VERSION=1.1
export NOVA_VERSION
OS_PASSWORD=baaadf76e15146e66201b8c6ba2f21d1287633f1
export OS_PASSWORD
OS_AUTH_TYPE=password
export OS_AUTH_TYPE
OS_AUTH_URL=http://192.168.25.1:5000/
export OS_AUTH_URL
OS_USERNAME=admin
OS_PROJECT_NAME=admin
COMPUTE_API_VERSION=1.1
OS_NO_CACHE=True
OS_CLOUDNAME=undercloud
export OS_USERNAME
export OS_PROJECT_NAME
export COMPUTE_API_VERSION
export OS_NO_CACHE
export OS_CLOUDNAME
OS_IDENTITY_API_VERSION='3'
export OS_IDENTITY_API_VERSION
OS_PROJECT_DOMAIN_NAME='Default'
export OS_PROJECT_DOMAIN_NAME
OS_USER_DOMAIN_NAME='Default'
export OS_USER_DOMAIN_NAME

# Add OS_CLOUDNAME to PS1
if [ -z "${CLOUDPROMPT_ENABLED:-}" ]; then
    export PS1=${PS1:-""}
    export PS1=\${OS_CLOUDNAME:+"(\$OS_CLOUDNAME)"}\ $PS1
    export CLOUDPROMPT_ENABLED=1
fi

**Comment by venkataanil** [comment]
PLease try with OSP16

**Comment by asyedham** [comment]
@venkataanil Able to deploy undercloud with osp16 after applying these changes https://github.com/redhat-performance/jetpack/pull/164

**Comment by asyedham** [comment]
Closing this as it is fixed in #164
